src/test/jtx/impl/WSClient.cpp

```diff
@@ -117,7 +117,9 @@ class WSClientImpl : public WSClient
             error_code ec;
             if (!peerClosed_)
             {
-                ws_.async_close({}, [&](error_code ec) { stream_.close(ec); });
+                ws_.async_close({}, strand_.wrap([&](error_code ec) {
+                    stream_.close(ec);
```

Comment:
It is not necessary to call `stream_.close(ec)` as its already closed with `ws_async_close`. However, calling `stream_.cancel(ec)` is a good idea to stop all asynchronous operations associated with the socket.

Reply:
Fixed and pushed.